tests/test_lib.py

```diff
@@ -22,6 +22,7 @@
 class TestUpdateJavaVersion(ccmtest.Tester):
     env = dict()
     temp_dir = None
+    all_versions = ['2.2', '3.0', '3.1', '3.11', '4.0', '4.1', '5.0', '5.1']
```

Comment:
where we hardcode version numbers like this, we need to make note of it in our release docs.
(the release manager needs to know all the places that they follow up changes. it's impossible to remember, and fixing breakages one-by-one can last days…)

Reply:
this is just a unit test and we do not need to update it; it just a set of versions which supports different java versions